src/sentry/ingest/consumer/processors.py

```diff
@@ -202,6 +203,10 @@ def process_event(
         else:
             with metrics.timer("ingest_consumer._store_event"):
                 cache_key = processing_store.store(data)
+            track_sampled_event(
+                data["event_id"], data.get("type"), TransactionStageStatus.REDIS_PUT
```

Comment:
I think you want the pipeline name not `data.get("type")` here. They are not always the same.

Reply:
im using `data.get("type")` to make it generalized, so when this is eventually extended for errors, it will work too. Do you have any concerns a bout this?

Reply:
hardcoded it to only take transactions for now

Reply:
Yes, you cannot use `data.get("type")` in errors - there are many different types going through the errors pipeline. The pipeline name should be the generalized thing.